package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "start": "webpack-dev-server --progress",
     "bundle": "webpack --mode production",
-    "build": "bundle install && npm install && npm run bundle",
+    "build": "bundle install && npm install && bundle exec rails assets:precompile && npm run bundle",
```

Comment:
npm install should be npm ci

Can you also adjust the Dockerfile so it doesn’t duplicate on running these commands?

Reply:
Done :+1: